src/maheshKJun20/ArrayExample/AllIndexWithDifferenceInStringArray.java

```diff
@@ -0,0 +1,37 @@
+/*
+Find all differ index from given arrays.
+i/p:- arr1: {"Maulik", "Harsh", "Krishna", "Technocredits"}
+arr2: {"Maulik","Harsh","Technocredits","Krishna"}		
+o/p:- Values are not matching at index -> 2, [arr1 value is Krishna, arr2 value is Technocredits] 
+Values are not matching at index -> 3, [arr1 value is Technocredits, arr2 value is Krishna]
+ */
+
+package maheshKJun20.ArrayExample;
+
+public class AllIndexWithDifferenceInStringArray {
+	
+	void displayIndexValueWithDiff(String[] strArray1, String[] strArray2) {
+		//Method to display difference in value of each element in String Array. It will also validate if length of 2 arrays is equal before comparison
+		boolean flag = true;
+		if(strArray1.length == strArray2.length) {
+			for (int index=0; index<strArray1.length;index++) {
+				if (!strArray1[index].equalsIgnoreCase(strArray2[index])) {
+					System.out.println("Values are not matching at index -> " + index + ", [strArray1 value is " + strArray1[index] + ", strArray2 value is " + strArray2[index] +"]" );
+					flag = false;
+				}				
+			}
+			if (flag)
+				System.out.println("Values of all Elements in both Array are equal");
```

Comment:
This if block is not required because displayIndexValueWithDiff() method name is this.

Reply:
Updated the Code as per above comments